src/azul/terraform.py

```diff
@@ -921,6 +937,39 @@ def tf_config(self, app_name):
                 sqs_name, _ = config.unqualified_resource_name(resource_name, suffix)
                 resource['event_source_arn'] = f'${{aws_sqs_queue.{sqs_name}.arn}}'
 
+        # Replace the references to unqualified Lambda function ARNs emitted by
+        # Chalice with references to the Lambda function alias.
+        #
+        locals[app_name] = re.sub(r'\$\{aws_lambda_function\.([^\.\s]+)\.invoke_arn\}',
```

Comment:
```suggestion
        locals[app_name] = re.sub(r'\$\{aws_lambda_function\.([^.\s]+)\.invoke_arn\}',
```

Also, the exclusion of whitespace makes no sense because you require a literal `.` to appear afterwards, so if whitespace terminated the `[…]+` match, that literal dot wouldn't match.